common/scala/src/main/scala/whisk/common/TransactionId.scala

```diff
@@ -25,7 +25,7 @@ import java.util.concurrent.atomic.AtomicInteger
 import scala.math.BigDecimal.int2bigDecimal
 import scala.util.Try
 
-import akka.event.Logging.{ InfoLevel, WarningLevel }
+import akka.event.Logging.{InfoLevel, WarningLevel}
```

Comment:
Is there a way to force these imports to be separate? I'm not sure how others feel, but I like to have imports not use wildcards or have multiples on same line.

Reply:
Hm, I'd argue against this. Wildcards are very handy especially in case of implicit imports.

Reply:
Ok, ignoring wildcards, WDYT about changing these to separate imports? I think the `ExpandImportSelectors` rewrite in scalafmt will do it (while leaving wildcards alone) http://scalameta.org/scalafmt/#ExpandImportSelectors. 

I think it will provide a good change to get imports formatted consistently (along with `SortedImported`).